<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formcreateresource</class>
 <widget class="QWidget" name="formcreateresource">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создание ресурса</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Выберите тип ресурсов</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbTypes">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать тип ресурсов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fldResName">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать тип ресурсов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
